app/views/organisations/_featured_news_no10.html.erb

```diff
@@ -17,8 +17,17 @@
       } %>
     </div>
   </div>
-
-  <% @documents.remaining_featured_news.first(6).in_groups_of(3, false) do |news| %>
+  <%
+    first_document = @documents.remaining_featured_news.first
```

Comment:
Thanks @leenagupte - for some reason I can't get it to work, it seems that `.shift` does return the first element, but it doesn't cause `@documents.remaining_featured_news` to change. Would you know why?

Reply:
I'm not sure why `shift` isn't modifying the variable. Perhaps because it's an instance variable.

However, I had a look at the code, and there's already a `first_featured_news` method that gets the first news item and merges `large: true` into it.

Perhaps we can use the existing methods if they work ok? Could you take a look and see if that's a viable option?

I've branched off your changes: https://github.com/alphagov/collections/pull/3797

https://collections-pr-3797.herokuapp.com/government/organisations/prime-ministers-office-10-downing-street

Reply:
Thanks @leenagupte - that looks good to me, so I've merged your commit into this branch 👍